openfish-webapp/src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
Is this really needed?

Reply:
It is part of the project starter template, I don't know what it does

Reply:
Ok, I've found an explanation: https://vitejs.dev/guide/features.html#client-types 
It adds some type definitions for typescript